.husky/pre-push

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+. "$(dirname -- "$0")/common.sh"
+
+yarn run build
+yarn run test:integration
```

Comment:
let's not have a pre-push for now, can remove this file

Reply:
removed